### PR TITLE
IMB-32 (Personal details) and  IMB-18 (Save and Exit)

### PR DESCRIPTION
**Pull request by TemitopeAyokuHO**
## What
[IMB-18](https://collaboration.homeoffice.gov.uk/jira/browse/IMB-18). (Save and Exit)

[IMB-32](https://collaboration.homeoffice.gov.uk/jira/browse/IMB-32). (Personal details)

## why

To excuse IMA figma flow

## How
Added different pages as shown on Figma.

Add url path and declare fields name in Ima/index.js.
created field for every input component in field.js.
Add Labels and text for each pages also created template for large text.
Added Template

Create a template on Gov notify for Save and Exit email once user exit and return to the application.
Add the template Id to config.js file and make a reference in .env file SAVE_AND_EXIT_TEMPLATEID.
Functionality for save and exit

Added a function to send email to applicant which contains new url in the save-form-session.js
## Test
Tested locally

### CODE REVIEW

File: apps/ima/translations/src/en/fields.json [hunk 1]
```diff
@@ -3,19 +3,162 @@
     "legend": "Which claim do you want to continue?"
   },
   "who-are-you": {
-    "legend": "Who are you?",
```
Comment (Rhodine-orleans-lindsay) [inline apps/ima/translations/src/en/fields.json:6]: keep this in

File: apps/ima/translations/src/en/pages.json [hunk 2]
```diff
@@ -8,37 +8,86 @@
...
+  "helper-details":{
+    "header":"Helper details",
+    "paragraph":"Answer all the following questions as if you are the person on the Illegal Migration Act notice."
+  },
+  "full-name":{ 
+    "header": "What is your full name?"
+  },
+  "who-are-you":{ 
+    "header": "Who are you?"
+  },
```
Comment (Rhodine-orleans-lindsay) [inline apps/ima/translations/src/en/pages.json:23]: remove this

File: apps/ima/views/content/cannot-use-form.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<span>You can only use this form if you are in the UK.</span>
```
Comment (Rhodine-orleans-lindsay) [inline apps/ima/views/content/cannot-use-form.md:1]: you don't need the span tag

File: apps/ima/views/save-and-exit.html [hunk 4]
```diff
@@ -1,5 +1,9 @@
 {{<partials-page}}
   {{$page-content}}
     <p>{{#t}}pages.save-and-exit.paragraph1{{/t}}</p>
+
+    <div class="other-details">
```
Comment (Rhodine-orleans-lindsay) [inline apps/ima/views/save-and-exit.html:5]: other-details class has no styling in css so this div and class is not needed and can be removed

File: apps/ima/views/helper-details.html [hunk 5]
```diff
@@ -0,0 +1,15 @@
+{{<partials-page}}
+  {{$page-content}}
+  {{#input-text}}someone-else-fullname{{/input-text}}
+  {{#input-text}}someone-else-relationship{{/input-text}} 
+  {{#input-text}}someone-else-organisation{{/input-text}}
+   
+  <p class="other-details">
```
Comment (Rhodine-orleans-lindsay) [inline apps/ima/views/helper-details.html:7]: remove other-details class

**Review by Rhodine-orleans-lindsay — CHANGES_REQUESTED** [review]
See comments

File: apps/ima/behaviours/save-form-session.js [hunk 6]
```diff
@@ -1,8 +1,23 @@
 /* eslint-disable camelcase */
-
```
Comment (Rhodine-orleans-lindsay) [inline apps/ima/behaviours/save-form-session.js:2]: keep the space

**Review by Rhodine-orleans-lindsay — CHANGES_REQUESTED** [review]
See comments. there is also a .DS_Store that has been added to the could you remove this please? When you done could you squash your commits please also make clear what changes apply to what ticket as there are two tickets in one PR  e.g. 

> IMB-32 (Personal details pages) and IMB-18 (Save and Exit pages)
> - IMB-18 (Save and Exit)

- >  - bullet point 1
- >  - bullet point 2, etc

> - IMB-32 (Personal details)

- >  - bullet point 1
- >  - bullet point 2, etc

**Review by Rhodine-orleans-lindsay — APPROVED** [review]

**Comment by TemitopeAyokuHO** [comment]
Test url -  ima-imb-18-and-imb-32.internal.branch.sas-notprod.homeoffice.gov.uk

**Comment by TemitopeAyokuHO** [comment]
Test confirmed ok

**Pull request merged**